CBLClient/Apps/CBLTestServer-iOS/CBLTestServer-iOS/Server/PredictiveQueriesRequestHandler.swift

```diff
@@ -0,0 +1,141 @@
+//
+//  PredictiveQueriesRequestHandler.swift
+//  CBLTestServer-iOS
+//
+//  Created by Sridevi Saragadam on 2/20/19.
+//  Copyright © 2017 Sridevi Saragadam. All rights reserved.
+//
+
+import Foundation
+import CouchbaseLiteSwift
+
+
+public class PredictiveQueriesRequestHandler {
+    public static let VOID: String? = nil
+    fileprivate var _pushPullReplListener:NSObjectProtocol?
+    
+    public func handleRequest(method: String, args: Args) throws -> Any? {
+        switch method {
+            
+        case "predictiveQuery_registerModel":
+            let modelName: String = args.get(name: "model_name")!
+            let echoModel = EchoModel(name: modelName)
```

Comment:
var type is missing, is it intended?

Reply:
var is used if you reassigning the variable. let is kind of constant, if you do not reassign the value, let has to be used